Mention tgswitch in recommended_tools.md (#128)
gruntwork-io/docs: _docs-sources/intro/environment-setup/recommended_tools.md

* [Terraform](https://www.terraform.io/downloads) – Terraform's declarative nature describes your infrastructure as code. If you're using Gruntwork's products, you're using Terraform.
* [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
* [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.

> * [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` was inspired by a similar tool `rbenv` for Ruby versions. The tool is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform.

Author: Does this read better?

```suggestion
* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
```

> * [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for versioning Terragrunt written in golang that offers similar features as `tfenv`, including managing the versions to use in a version file.

Reviewer: ```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

Reviewer: Not sure if that reads a little better?

Author: Thanks Eugene, I'll just make a slightly more improved suggestion

```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

> * [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.

Author: ```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch#installation) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```
